shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/ddl/DDLStatementAssert.java

```diff
@@ -72,6 +75,8 @@ public static void assertIs(final SQLCaseAssertContext assertContext, final DDLS
             AlterIndexStatementAssert.assertIs(assertContext, (AlterIndexStatement) actual, (AlterIndexStatementTestCase) expected);
         } else if (actual instanceof DropIndexStatement) {
             DropIndexStatementAssert.assertIs(assertContext, (DropIndexStatement) actual, (DropIndexStatementTestCase) expected);
+        } else if (actual instanceof OracleAlterSynonymStatement) {
```

Comment:
Do we need to test OracleAlterSynonymStatement, as we are only interested in table-related and index-related Statements? @tristaZero

Reply:
Hi sorry for my late comment.
Is `AlterSynonym` similar with `AlterTable`? 
If so, OracleAlterSynonymStatement is needed, I suppose.

Reply:
Hi @tristaZero @wgy8283335

If we are only interested in table-related and index-related Statements, then we probably don't need to test `OracleAlterSynonymStatement`.

But when I looked at dcl and here basically all statements like for example `AlterRoleStatement`, `AlterUserStatement` are tested. So I thought that we also need it for `OracleAlterSynonymStatement`. 

Should I delete it or keep it?

Reply:
Keep it.

Reply:
Okay:) thank you